modules/backend/classes/NavigationManager.php

```diff
@@ -1,7 +1,9 @@
 <?php namespace Backend\Classes;
 
+use Backend\Models\User;
```

Comment:
Why is this being included?

Reply:
I've changed the documentation for the method `filterItemPermissions` to properly reference the User class. Then my IDE imported it. I've removed the import and modified the documentation to reference the full namespace.